src/components/tests/CaptureFilter.test.js

```diff
@@ -0,0 +1,182 @@
+        expect(input).toBeInTheDocument();
+      });
+
+      it('renders Species dropdown ', () => {
+        render(component);
+        let dropdown = screen.getByTestId('species-dropdown');
+        expect(dropdown).toBeInTheDocument();
+      });
+
+      it('renders Tags dropdown ', () => {
+        render(component);
+        let dropdown = screen.getByTestId('tag-dropdown');
+        expect(dropdown).toBeInTheDocument();
+      });
+
+      it('renders Organization dropdown ', () => {
+        render(component);
+        let dropdown = screen.getByTestId('org-dropdown');
+        expect(dropdown).toBeInTheDocument();
+      });
+
+      it('renders default orgList when dropdown clicked ', () => {
+        render(component);
+        let dropdown = screen.getByTestId('org-dropdown');
+        expect(dropdown).toBeInTheDocument();
+
+        let button = within(dropdown).getByRole('button', {
+          name: /all/i,
+        });
+
+        userEvent.click(button);
+
+        // the actual list of orgs is displayed in a popup that is not part of CaptureFilter
+        // this list is the default list
+        const orglist = screen.getByRole('listbox');
+        const orgs = within(orglist).getAllByTestId('org-item');
+        const listItems = orgs.map((org) => org.textContent);
+        console.log('default orgList', listItems);
+
+        expect(orgs).toHaveLength(2);
+      });
+    });
+
+    describe('w/ data in context', () => {
+      let orgs;
+      let component;
+
+      beforeEach(async () => {
+        orgs = await api.getOrganizations();
+        component = (
+          <AppProvider value={{ orgList: orgs }}>
+            <CaptureFilter />
+          </AppProvider>
+        );
+        // render(component);
+        await act(() => api.getOrganizations());
+      });
+
+      afterEach(cleanup);
+
+      it('api loaded 2 organizations', () => {
+        expect(orgs).toHaveLength(2);
+      });
+
+      it('renders Organization dropdown ', () => {
+        render(component);
+        let dropdown = screen.getByTestId('org-dropdown');
+        expect(dropdown).toBeInTheDocument();
+      });
+
+      it('renders default orgList when dropdown clicked ', () => {
+        render(component);
+        let dropdown = screen.getByTestId('org-dropdown');
+        expect(dropdown).toBeInTheDocument();
+
+        let button = within(dropdown).getByRole('button', { name: /all/i });
+
+        userEvent.click(button);
+
+        // screen.logTestingPlaygroundURL();
+
+        // the actual list of orgs is displayed in a popup that is not part of CaptureFilter
+        const orglist = screen.getByRole('listbox');
+        const orgs = within(orglist).getAllByTestId('org-item');
+        const listItems = orgs.map((org) => org.textContent);
+        console.log('default orgList', listItems);
+
+        expect(orgs).toHaveLength(2);
+      });
+    });
+
+    // describe('context data renders in child', () => {
```

Comment:
Is this test awaiting implementation? Rather than comment it out, can it be either removed or marked `skip`?